third-party/realsense-file/rosbag/msgs/sensor_msgs/Image.h

```diff
@@ -238,6 +242,14 @@ namespace serialization
       stream.next(m.is_bigendian);
       stream.next(m.step);
       stream.next(m.data);
+      try
+      {
+          stream.next(m.depth_units);
```

Comment:
There is a header that should be used to differentiate according to version number